treescript/src/treescript/merges.py

```diff
@@ -25,20 +25,23 @@ def replace(file_name, from_, to_):
         f.write(new_text)
 
 
-def touch_clobber_file(repo_path):
+def touch_clobber_file(config, repo_path):
     """Update the clobber file in the root of the repo."""
-    log.info("Touching clobber file")
-    clobber_file = os.path.join(repo_path, "CLOBBER")
-    with open(clobber_file) as f:
-        contents = f.read()
-    new_contents = ""
-    for line in contents.splitlines():
-        line = line.strip()
-        if line.startswith("#") or line == "":
-            new_contents += f"{line}\n"
-    new_contents += "Merge day clobber"
-    with open(clobber_file, "w") as f:
-        f.write(new_contents)
+    if config["merge_day_clobber_file"]:
+        log.info("Touching clobber file")
+        clobber_file = os.path.join(repo_path, config["merge_day_clobber_file"])
+        with open(clobber_file) as f:
+            contents = f.read()
+        new_contents = ""
+        for line in contents.splitlines():
+            line = line.strip()
+            if line.startswith("#") or line == "":
+                new_contents += f"{line}\n"
+        new_contents += "Merge day clobber"
+        with open(clobber_file, "w") as f:
+            f.write(new_contents)
+    else:
+        log.info("merge_day_clobber_file not set, skipping clobber file update")
```

Comment:
Minor preference: To me it looks better if we do `if not config[...` and return straight away as the `else` here ends up being quite far from the `if`. Not a blocking comment, though.

Reply:
Changed.